packages/fresh/pubspec.yaml

```diff
@@ -7,8 +7,8 @@ homepage: https://github.com/felangel/fresh
 version: 0.4.1
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.19.0 <3.0.0"
 
 dev_dependencies:
   mocktail: ^0.3.0
-  test: ^1.16.0
+  test: ^1.23.1
```

Comment:
Please don't do this unless it is really needed. This breaks backwards compatibility with older Flutter versions.